projects/plugins/jetpack/_inc/client/search/dashboard/module-control.jsx

```diff
@@ -80,6 +83,33 @@ function Search( props ) {
 	const togglingInstantSearch = !! props.isSavingAnyOption( 'instant_search_enabled' );
 	const isSavingEitherOption = togglingModule || togglingInstantSearch;
 
+	const isInstantSearchCFAButtonDisabled =
```

Comment:
What's CFA? :)

Reply:
good question, working on it.

Reply:
"Call for action", maybe? 🙂

Reply:
That was my guess too :) Seems superfluous here and will just leave future developers pondering the same thing I think. Can we use `isInstantSearchButtonDisabled`?

Reply:
You two are both right - yep people seem to use `Call to action` more. Removed to avoid confusion.